packages/react-vapor/src/components/datePicker/DatePicker.tsx

```diff
@@ -39,10 +42,24 @@ export class DatePicker extends React.PureComponent<IDatePickerProps, {isSelecte
 
     private setToToday = () => {
         const date = new Date();
-        this.dateInput.value = this.getStringFromDate(date);
+        this.dateInput.value = this.props.minimalRangeLimit
+            ? this.getStringFromDate(this.updateDateBasedOnMinimalLimit(date))
+            : this.getStringFromDate(date);
         this.handleChangeDate();
     };
 
+    private updateDateBasedOnMinimalLimit = (date: Date) => {
+        const {weeks, days, hours, minutes} = this.props.minimalRangeLimit;
+        const minimalLimitInMinutes: number =
+            (weeks ? weeks * 10080 : 0) + (days ? days * 1440 : 0) + (hours ? hours * 60 : 0) + (minutes ? minutes : 0);
+        const diff = moment().diff(moment(date), 'minutes');
```

Comment:
This code seem similar with the one in DatePickerDropdown.tsx. 

Maybe that could be extracted into a DatePickerHelper?